src/modules/display.js

```diff
@@ -0,0 +1,48 @@
+import {
+  clearPreviousDOM, createMovieElements, getAllMovies, likeMovie,
+} from './utils.js';
+import movieCounter from './movieCounter.js';
+import { getAllLikes } from './API.js';
+
+export const displayMovies = (movies, events) => {
+  clearPreviousDOM();
+  movies.forEach((movie) => {
+    const { image, name } = movie.show;
+    const list = document.createElement('li');
+    list.setAttribute('class', 'movie-list');
+    const arrayElements = createMovieElements(image, name, movie.show);
+    arrayElements.forEach((elem) => {
+      list.appendChild(elem);
+    });
+    events.appendChild(list);
+  });
+  const numberOfMovies = movieCounter();
+  const movieHolderCount = document.querySelector('#movie_count');
+  movieHolderCount.innerHTML = `Number of movies  (${numberOfMovies})`;
+};
+
+export const displayLikes = (array) => {
+  const movieElements = document.querySelectorAll('.span-likes');
+  movieElements.forEach((element) => {
+    const index = element.getAttribute('data-index');
+    const ind = array.find((arr) => arr.item_id === index);
+    const par = document.createElement('p');
+    par.textContent = `${ind ? ind.likes : '0'} likes`;
+    element.appendChild(par);
+  });
```

Comment:
- [ ] Let's add tests for the likes counter as well, you can test for a situation where no like has been recorded and vice versa

Reply:
Please this is not a requirement to add the test for the like counter we have only comment counter and movies counter